packages/cli/src/commands/git/connect.ts

```diff
@@ -0,0 +1,165 @@
+import chalk from 'chalk';
+import { join } from 'path';
+import { Org, Project } from '../../types';
+import Client from '../../util/client';
+import { parseGitConfig, pluckRemoteUrl } from '../../util/create-git-meta';
+import confirm from '../../util/input/confirm';
+import { Output } from '../../util/output';
+import link from '../../util/output/link';
+import { getCommandName } from '../../util/pkg-name';
+import {
+  connectGitProvider,
+  disconnectGitProvider,
+  formatProvider,
+  parseRepoUrl,
+} from '../../util/projects/connect-git-provider';
+import validatePaths from '../../util/validate-paths';
+
+export default async function connect(
+  client: Client,
+  argv: any,
+  args: string[],
+  project: Project | undefined,
+  org: Org | undefined
+) {
+  const { output } = client;
+  const confirm = Boolean(argv['--confirm']);
+
+  if (args.length !== 0) {
+    output.error(
+      `Invalid number of arguments. Usage: ${chalk.cyan(
+        `${getCommandName('project connect')}`
+      )}`
+    );
+    return 2;
+  }
+  if (!project || !org) {
+    output.error(
+      `Can't find \`org\` or \`project\`. Make sure your current directory is linked to a Vercel projet by running ${getCommandName(
+        'link'
+      )}.`
+    );
+    return 1;
+  }
+
+  let paths = [process.cwd()];
+
+  const validate = await validatePaths(client, paths);
+  if (!validate.valid) {
+    return validate.exitCode;
+  }
+  const { path } = validate;
+
+  const gitProviderLink = project.link;
+  client.config.currentTeam = org.type === 'team' ? org.id : undefined;
+
+  // get project from .git
+  const gitConfigPath = join(path, '.git/config');
+  const gitConfig = await parseGitConfig(gitConfigPath, output);
+  if (!gitConfig) {
+    output.error(
+      `No local git repo found. Run ${chalk.cyan(
+        '`git clone <url>`'
+      )} to clone a remote Git repository first.`
+    );
+    return 1;
+  }
+  const remoteUrl = pluckRemoteUrl(gitConfig);
+  if (!remoteUrl) {
+    output.error(
+      `No remote origin URL found in your Git config. Make sure you've connected your local Git repo to a Git provider first.`
```

Comment:
```suggestion
      `No remote origin URL found in your Git config. Make sure you've configured a remote repo in your local git config. Run ${chalk.cyan('`git remote --help`')} for more details.`
```

I think using the word "connected" here might be confusing in an error message after the user tried to `vc git connect`. What do you think about this?